app/dashboard/templates/bounty/change.html

```diff
@@ -21,6 +21,7 @@
   {% include 'shared/head.html' %}
   {% include 'shared/cards.html' %}
   <link rel="stylesheet" href={% static "v2/css/submit_bounty.css" %}>
+  <link rel="stylesheet" href="{% static "v2/css/jquery.select2.min.css" %}">
```

Comment:
These css and js files are already loaded in `head.html` and `footer_scripts.html`